### ISSUE TITLE
Null Argument Expcetion Raise when navigating to a ContentPage that has a AutoSuggestBox

### ISSUE THREAD

**Issue by OlteanuRadu** [dotMorten/XamarinFormsControls#29]
Hello,

I'm trying to use the AutoSuggestBox in the following manner : 

<ScrollView VerticalOptions="StartAndExpand">
            <StackLayout Margin="10" Padding="10">
                <Label Text="Welcome to Xamarin.Forms!"
                VerticalOptions="CenterAndExpand" 
                HorizontalOptions="CenterAndExpand" />

                <dm:AutoSuggestBox x:Name="test1"
                           SuggestionChosen="V_SuggestionChosen"
                           QuerySubmitted="V_QuerySubmitted"
                        TextChanged="V_TextChanged"/>
            </StackLayout>
        </ScrollView>

and when I'm tring to navigate to this page, using the Navigation Service, I receive a NullArgumentException. 

Here is the stack trace; 

```
  at Foundation.NSAttributedString..ctor (System.String str, Foundation.NSDictionary attributes) [0x0000e] in /Library/Frameworks/Xamarin.iOS.framework/Versions/12.2.1.13/src/Xamarin.iOS/Foundation/NSAttributedString.g.cs:152 
  at Foundation.NSAttributedString..ctor (System.String str, UIKit.UIFont font, UIKit.UIColor foregroundColor, UIKit.UIColor backgroundColor, UIKit.UIColor strokeColor, UIKit.NSParagraphStyle paragraphStyle, Foundation.NSLigatureType ligatures, System.Single kerning, Foundation.NSUnderlineStyle underlineStyle, UIKit.NSShadow shadow, System.Single strokeWidth, Foundation.NSUnderlineStyle strikethroughStyle) [0x00000] in /Library/Frameworks/Xamarin.iOS.framework/Versions/12.2.1.13/src/Xamarin.iOS/Foundation/NSAttributedString.cs:158 
  at dotMorten.Xamarin.Forms.NativeAutoSuggestBox.SetPlaceholderTextColor (Xamarin.Forms.Color color) [0x00000] in E:\GitHub\dotMorten\XamarinFormsControls\AutoSuggestBox\dotMorten.Xamarin.Forms.AutoSuggestBox\NativeAutoSuggestBox.iOS.cs:108 
  at dotMorten.Xamarin.Forms.AutoSuggestBoxRenderer.UpdatePlaceholderTextColor () [0x0000c] in E:\GitHub\dotMorten\XamarinFormsControls\AutoSuggestBox\dotMorten.Xamarin.Forms.AutoSuggestBox\AutoSuggestBoxRenderer.cs:195 
  at dotMorten.Xamarin.Forms.AutoSuggestBoxRenderer.OnElementChanged (Xamarin.Forms.Platform.iOS.ElementChangedEventArgs`1[TElement] e) [0x000ea] in E:\GitHub\dotMorten\XamarinFormsControls\AutoSuggestBox\dotMorten.Xamarin.Forms.AutoSuggestBox\AutoSuggestBoxRenderer.cs:87 
  at Xamarin.Forms.Platform.iOS.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x0012a] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.iOS.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00000] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x00014] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementPackager.OnChildAdded (Xamarin.Forms.VisualElement view) [0x0003d] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementPackager.Load () [0x0001e] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000de] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.iOS.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00000] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x00014] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementPackager.OnChildAdded (Xamarin.Forms.VisualElement view) [0x0003d] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementPackager.Load () [0x0001e] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.ScrollViewRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00087] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x00014] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementPackager.OnChildAdded (Xamarin.Forms.VisualElement view) [0x0003d] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementPackager.Load () [0x0001e] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000de] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.iOS.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00000] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x00014] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementPackager.OnChildAdded (Xamarin.Forms.VisualElement view) [0x0003d] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.VisualElementPackager.Load () [0x0001e] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.PageRenderer.ViewDidLoad () [0x00086] in <18e93b810a4a4e2f919016f742528d37>:0 
  at (wrapper managed-to-native) ObjCRuntime.Messaging.IntPtr_objc_msgSendSuper(intptr,intptr)
  at UIKit.UIViewController.get_View () [0x0002a] in /Library/Frameworks/Xamarin.iOS.framework/Versions/12.2.1.13/src/Xamarin.iOS/UIKit/UIViewController.g.cs:2983 
  at Xamarin.Forms.Platform.iOS.PageRenderer.get_NativeView () [0x00008] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.PageRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x0004f] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x00014] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.NavigationRenderer.CreateViewControllerForPage (Xamarin.Forms.Page page) [0x00008] in <18e93b810a4a4e2f919016f742528d37>:0 
  at Xamarin.Forms.Platform.iOS.NavigationRenderer+<OnPushAsync>d__51.MoveNext () [0x0001d] in <18e93b810a4a4e2f919016f742528d37>:0 
--- End of stack trace from previous location where exception was thrown ---
  at Xamarin.Forms.NavigationPage+<PushAsyncInner>d__84.MoveNext () [0x00085] in D:\a\1\s\Xamarin.Forms.Core\NavigationPage.cs:394 
--- End of stack trace from previous location where exception was thrown ---
  at Xamarin.Forms.NavigationPage+<PushAsync>d__55.MoveNext () [0x0014e] in D:\a\1\s\Xamarin.Forms.Core\NavigationPage.cs:219 
--- End of stack trace from previous location where exception was thrown ---
  at Substance.Services.NavigationService+<InternalNavigateToAsync>d__12.MoveNext () [0x00138] in /Users/user149779/Substance/Substance/Substance/Substance/Services/NavigationService.cs:106 
--- End of stack trace from previous location where exception was thrown ---
  at Substance.ViewModels.FishingSessionAddViewModel+<<get_ViewOnMapCommand>b__43_0>d.MoveNext () [0x0002a] in /Users/user149779/Substance/Substance/Substance/Substance/ViewModels/FishingSessionAddViewModel.cs:181 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0 (System.Object state) [0x00000] in /Library/Frameworks/Xamarin.iOS.framework/Versions/12.2.1.13/src/Xamarin.iOS/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/AsyncMethodBuilder.cs:1023 
  at Foundation.NSAsyncSynchronizationContextDispatcher.Apply () [0x00000] in /Library/Frameworks/Xamarin.iOS.framework/Versions/12.2.1.13/src/Xamarin.iOS/Foundation/NSAction.cs:178 
--- End of stack trace from previous location where exception was thrown ---
  at (wrapper managed-to-native) UIKit.UIApplication.UIApplicationMain(int,string[],intptr,intptr)
  at UIKit.UIApplication.Main (System.String[] args, System.IntPtr principal, System.IntPtr delegate) [0x00005] in /Library/Frameworks/Xamarin.iOS.framework/Versions/12.2.1.13/src/Xamarin.iOS/UIKit/UIApplication.cs:79 
  at UIKit.UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName) [0x0002c] in /Library/Frameworks/Xamarin.iOS.framework/Versions/12.2.1.13/src/Xamarin.iOS/UIKit/UIApplication.cs:63 
  at Substance.iOS.Application.Main (System.String[] args) [0x00001] in /Users/user149779/Substance/Substance/Substance/Substance.iOS/Main.cs:16
```

**Comment by dotMorten** [comment]
Could you provide a simple repro sample to help me reproduce this issue z as I'm not seeing it during page navigation.

**Comment by dotMorten** [comment]
I think I got the repro: It happens when the placeholder is empty

**Comment by dotMorten** [comment]
Fixed in v0.5.0
